framework/auth/views.py

```diff
@@ -248,13 +359,15 @@ def send_confirm_email(user, email):
         merge_target = None
 
     campaign = campaigns.campaign_for_user(user)
-    # Choose the appropriate email template to use
+    # Choose the appropriate email template to use and add existing_user flag if a merge or adding an email.
     if merge_target:
         mail_template = mails.CONFIRM_MERGE
+        confirmation_url += '?existing_user={}'.format(user.email)
```

Comment:
Is `confirmation_url` guaranteed not to have any other query params at this point?